src/main/resources/templates/fragments/header.html

```diff
@@ -4,8 +4,8 @@
         <img
             alt="Logo der Universität Luzern"
             class="m-4"
-            id="logo"
-            th:src="@{images/UNILU_Schriftzug_Standard_schwarz_DE.png}" />
+            id="unilu-logo-default"
```

Comment:
I'd rename this. Instead of telling me how it looks, tell why it looks like that. `-default` -> is the default size, which I can already see. `-header` -> this is the logo which I can find in the header.  
```suggestion
            id="unilu-logo-header"
 ```